source/overview/features.rst

```diff
@@ -0,0 +1,304 @@
+============================
+Features Overview
+============================
+
+This page gives an overview of some of the features available in Mattermost Team Edition and Enterprise Edition.
```

Comment:
I understand we're trying to separate Team and Enterprise editions by repeating the word "Edition" but it's a little clunky grammatically...

Propose: "...in Mattermost Team and Enterprise Editions." ?